media/css/custom.scss

```diff
@@ -471,3 +476,34 @@ summary {
 
 @import "faculty_structure";
 @import "honours_board";
+
+.relax-item {
+  width: 200px;
+  height: 200px;
+  background: lavenderblush;
```

Comment:
Color `lavenderblush` should be written in hexadecimal form as `#fff0f5`<br>Color literals like `lavenderblush` should only be used in variable declarations; they should be referred to via variable everywhere else.